mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -17,6 +21,8 @@ CLASS net/minecraft/class_66 net/minecraft/enchantment/EnchantmentHelper
 		ARG 1 stack
 	METHOD method_108 getEnchantments (Lnet/minecraft/class_2056;)Ljava/util/Map;
 		ARG 0 stack
+	METHOD method_110 EnchantmentResistance ([Lnet/minecraft/class_2056;Lnet/minecraft/class_1733;)I
```

Comment:
should this be `getEnchantmentResistance` maybe?

Reply:
most likely